example-app/src/main/java/backtraceio/backtraceio/MainActivity.java

```diff
@@ -68,6 +70,11 @@ protected void onCreate(Bundle savedInstanceState) {
 
     public native void cppCrash();
 
+    public native boolean registerNativeBreadcrumbs(BacktraceBase backtraceBase);
```

Comment:
Hey - I think it makes more sense to avoid adding/disabling breadcrumbs in the demo app and instead just enable it. What do you think?

Reply:
I think you're right, I just exposed it for testing.

Reply:
Maybe we should have a separate private test app or something

Reply:
What if other developer/tester wants to use this button later?

Reply:
I think you're right...we don't need to worry about disable.

Reply:
Removed `disableBreadcrumbs`, thank you!